lib/cosmos/io/win32_serial_driver.rb

```diff
@@ -8,15 +8,12 @@
 # as published by the Free Software Foundation; version 3 with
 # attribution addendums as found in the LICENSE.txt
 
-require 'cosmos/io/serial_driver'
```

Comment:
This was causing a circular require dependency. This file uses SerialDriver but it always requires this and I don't think this file should be used stand alone anyway.